code/script.js

```diff
@@ -235,20 +238,18 @@ const start = () => {
   charactersInPlay = CHARACTERS
   setSecret()
   generateBoard()
-  selectQuestion()
+  
   // Here we're setting charactersInPlay array to be all the characters to start with
   // What else should happen when we start the game?
 }
 
 // setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-  const value = questions.value
- 
+  const value = questions.options[questions.selectedIndex].value
```

Comment:
Another way to get the value is 
`  const value = questions.value;`

Because the all the `options `are values of `select`, in this case, it's unnecessary to attach the  `options[questions.selectedIndex]`